src/editor/EditorCommandHandlers.js

```diff
@@ -297,6 +297,22 @@ define(function (require, exports, module) {
         return false;
     }
 
+    /**
+     * Return the column of the first non whitespace char in the given line.
+     *
+     * @private
+     * @param {!Document} doc
+     * @param {number} lineNum
+     * @returns {number} the column index or null
+     */
+    function _firstNotWs(doc, lineNum) {
+        var text = doc.getLine(lineNum);
+        if (text === null || text === undefined) {
+            return null;
```

Comment:
The return value from this function is not validated -- it gets used directly as a char position. Seems like it would be more consistent with CodeMirror APIs to return `-1` here (but my CM API knowledge is rusty...).